app/helpers/form_helper.rb

```diff
@@ -316,8 +316,8 @@ def add_label options, f, attr
     label_size = options.delete(:label_size) || "col-md-2"
     required_mark = check_required(options, f, attr)
     label = ''.html_safe + options.delete(:label)
-    if label.empty? && f.try(:object) && ((clazz = f.object.class).respond_to?(:gettext_translation_for_attribute_name))
-      label = s_(clazz.gettext_translation_for_attribute_name attr).html_safe
+    if label.empty? && f.try(:object) && ((clazz =f.object.class).respond_to?(:gettext_translation_for_attribute_name))
```

Comment:
a small nit: missing a space in `(clazz =f.object.class)`. It should be `(clazz = f.object.class)`